services/user/CommonSys/ui/src/views/dashboard.tsx

```diff
@@ -1,16 +1,34 @@
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 
-import { AppletEntry, appletPrefix, applets } from "../config";
+import { siblingUrl } from "common/rpc.mjs";
+import { AppletEntry, applets } from "../config";
 import { Heading, Text } from "../components";
 
 export const Dashboard = ({ currentUser }: { currentUser: string }) => {
+    const [hrefs, setHrefs] = useState<{ [key: string]: string }>({});
+
     useEffect(() => {
         window.document.title = "Psibase Dashboard";
+
+        const fetchHrefs = async () => {
+            const newHrefs: { [key: string]: string } = {};
+
+            for (const applet of applets) {
+                newHrefs[applet.service] = await siblingUrl(
+                    null,
+                    applet.service
+                );
+            }
+
+            setHrefs(newHrefs);
```

Comment:
Could probably get away with a `Promise.all` here rather than one at a time, but no biggy, not worth holding this up. :+1: